bitnami/mongodb-sharded/templates/mongos/mongos-service.yaml

```
@@ -44,6 +44,11 @@ spec:
     - name: metrics
       port: 9216
       targetPort: metrics
+      {{- if .Values.service.nodePorts.mongodb }}
+      nodePort: {{ add .Values.service.nodePorts.mongodb 1 }}
+      {{- else if eq .Values.service.type "ClusterIP" }}
+      nodePort: null
+      {{- end }}
```

I think we should be more explicit and add a new value to set the metrics  `nodePort` to allow users to set directly the port. Just Imagine the scenario where the `service.nodePorts.mongodb + 1` port is currently assigned to other service or it is already in use in the node.

```suggestion
      {{- if .Values.service.nodePorts.metrics }}
      nodePort: {{ .Values.service.nodePorts.metrics }}
      {{- else if eq .Values.service.type "ClusterIP" }}
      nodePort: null
      {{- end }}
```